### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by wangfan860**
### Improvements
Add description for SSR notebook

### CODE REVIEW

**Review by paulineribeyre — DISMISSED** [review]

**Review by wangfan860 — COMMENTED** [review]
I think Gina want it to be `patient`😉https://cdis.slack.com/archives/CV4HHPB9Q/p1605807141211600?thread_ts=1605800488.208500&cid=CV4HHPB9Q

**Comment by paulineribeyre** [comment]
sorry, i got it backwards 🤪

File: covid19-notebooks/readme.md [hunk 1]
```diff
@@ -40,3 +40,15 @@ In this R markdown, we track COVID-19 local (US) and global cases with active, c
 ### "COVID-19 Testing Over Time in the US" notebook:
 
 COVID-19 testing is very crucial to understand the spread of the pandemic. In this python notebook, we have focused our efforts on visualizing the COVID-19 testing data across the U.S. States and Territories from the COVID Tracking Project. The animations will allow you to see how testing data has changed over time.
```
Comment (gkuffel) [inline covid19-notebooks/readme.md:42]: Change this line: "U.S. States and Territories" to United States and U.S. Territories

File: covid19-notebooks/readme.md [hunk 2]
```diff
@@ -40,3 +40,15 @@ In this R markdown, we track COVID-19 local (US) and global cases with active, c
...
+
+### "Cross Dataset Analysis on COVID-19 Symptoms and Fatality Rate" notebook:
+
+In this notebook, we are performing visual analysis on the individual case reports that provide aggregated data on clinical states (eg., deceased, recovered, alive, etc) and symptoms. We are looking at whether the fatality rates and symptoms are similar across different datasets.
+
+### "Demo: Exploring clinical data from multiple commons" notebook:
+
+In this notebook, we obtain the clinical data for multiple projects from Chicagoland Pandemic Response Commons and NCI Genomic Data Commons (GDC) and visualize some of the clinical attributes. This notebook is for demonstration purposes only and not biologically meaningful.
```
Comment (gkuffel) [inline covid19-notebooks/readme.md:50]: Add "the" in front of Chicagoland Pandemic Response

File: covid19-notebooks/readme.md [hunk 3]
```diff
@@ -40,3 +40,15 @@ In this R markdown, we track COVID-19 local (US) and global cases with active, c
...
+
+### "Visualize Statistical Summary Reports from Partnering Healthcare Providers" notebook:
+
+This notebook provides an example on how to visualize the status of COVID-19 and general patient information at multiple regional healthcare providers using the demo Statistical Summary Report data from 03/10/2020 to 11/10/2020.
```
Comment (gkuffel) [inline covid19-notebooks/readme.md:54]: Change "on" to "of".

Change "at" to  "from"

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**